FluentTc.Tests/ProjectRetrievalTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Collections.Generic;
+using FakeItEasy;
+using FluentTc.Domain;
+using FluentTc.Engine;
+using NUnit.Framework;
+
+namespace FluentTc.Tests
+{
+    [TestFixture]
+    public class ProjectRetrievalTests
+    {
+        [TestCase("FluentTC")]
+        [TestCase("Test 1")]
+        [TestCase("Test%20C")]
+        public void GetProject_ByName(string projectName)
```

Comment:
I was actually just testing random characters and thought that would be the most common that someone would enter. Currently, not handling URL encoding.